### PR TITLE
COPS-31: fix user for jstack

### PR DESCRIPTION
**Pull request by Ronan-Gorain**
JIRA issue: https://jira.jahia.org/browse/COPS-31

Short description:

### CODE REVIEW

File: thread-dump/thread-dump.yml [hunk 1]
```diff
@@ -4,6 +4,8 @@ version: 1.5.2
 name: thread dump
 id: enable-disable-thread-dump
 
+### !!! Do not work on jahia < 7.3 !!!
```
Comment (kevinvandel) [inline thread-dump/thread-dump.yml:7]: *Does :stuck_out_tongue:

Comment (Ronan-Gorain) [reply thread-dump/thread-dump.yml:7]: It's an order, I'm trying to persuade it

File: thread-dump/thread-dump.sh [hunk 2]
```diff
@@ -14,7 +20,13 @@ PID=$(ps awwx |grep catalina.startup.Bootstrap |grep -v grep |cut -c -5)
 hour=$(date +'%Hh')
 minute=$(date +'%M')
 mkdir -p "$REPO/$today/$hour"
-sudo -u tomcat jstack -l $PID > "$REPO/$today/$hour/$minute"
+
+version=$(grep DX_VERSION /.jelenv |cut -d'=' -f2)
```
Comment (laurentfufu) [inline thread-dump/thread-dump.sh:24]: I would love to see `version=$(awk -F'=' '$1=="DX_VERSION" {print $2}' /.jelenv)` here or at least use  `version=$(grep "^DX_VERSION$" /.jelenv | cut -d'=' -f2` :trollface:

File: thread-dump/thread-dump.sh [hunk 3]
```diff
@@ -14,7 +20,13 @@ PID=$(ps awwx |grep catalina.startup.Bootstrap |grep -v grep |cut -c -5)
...
+if [[ $version == 7.3* ]]; then
+    jstack -l $PID > "$REPO/$today/$hour/$minute"
+else
+    sudo -u tomcat jstack -l $PID > "$REPO/$today/$hour/$minute"
+fi
 
 # remove files older than 30 days
 find $REPO -type f -mtime +30 -exec rm -f {} \;
```
Comment (laurentfufu) [inline thread-dump/thread-dump.sh:32]: you remove old files but old folders still remain

**Comment by laurentfufu** [comment]
@kevinvandel  show me the path, sorry

File: thread-dump/thread-dump.sh (hunk 3 above)
Comment (Ronan-Gorain) [reply thread-dump/thread-dump.sh:32]: There should be no folder remaining at this point. It is already deleted at compression (line 15)

File: thread-dump/thread-dump.sh (hunk 2 above)
Comment (Ronan-Gorain) [reply thread-dump/thread-dump.sh:24]: You are right, fixed

File: thread-dump/thread-dump.sh (hunk 3 above)
Comment (laurentfufu) [reply thread-dump/thread-dump.sh:32]: you damn right, sorry !

**Pull request closed without merging**